### PR TITLE
Updating read_posts extension to correctly redim undimmed posts

### PR DESCRIPTION
**Pull request by blackjackkent**

### CODE REVIEW

File: Extensions/read_posts.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //* TITLE Read Posts **//
-//* VERSION 0.1 REV G **//
+//* VERSION 0.2 REV G **//
```
Comment (bvtsang) [inline Extensions/read_posts.js]: We're using [Semantic Versioning](http://semver.org/), so `REV G` isn't needed.

File: Extensions/read_posts.js [hunk 2]
```diff
@@ -20,37 +20,37 @@ XKit.extensions.read_posts = new Object({
 
 	dashboard_regex: new RegExp("^https?://www.tumblr.com/dashboard(?:$|/)\\#?"),
 
-	last_undimmed: "",
+	undimmed_post: null,
+        currently_undimming: false,
```
Comment (bvtsang) [inline Extensions/read_posts.js]: Nitpick: uneven indentation

Comment (blackjackkent) [inline Extensions/read_posts.js]: Oop. Meant to clean that before I pushed. *slaps own wrist* Will update; one sec.

File: Extensions/read_posts.js (hunk 1 above)
Comment (blackjackkent) [inline Extensions/read_posts.js]: Oop.

**Comment by blackjackkent** [comment]
Updated to fix above comments. :)

File: Extensions/read_posts.js [hunk 3]
```diff
@@ -20,51 +20,48 @@ XKit.extensions.read_posts = new Object({
...
-		// This is broken and meh.
-		var m_obj = $(XKit.extensions.read_posts.last_dimmed).find(".post");
-		if (!$(XKit.extensions.read_posts.last_dimmed).hasClass("post")) {
-			var m_obj = $(m_obj).parentsUntil('.post').parent();
-		}
-		if (!$(m_obj).hasClass("post")) {
-			// STILL?!
-			return;
+		// This is less broken and meh.
+		if (XKit.extensions.read_posts.undimmed_post == null || XKit.extensions.read_posts.currently_undimming) {
```
Comment (bvtsang) [inline Extensions/read_posts.js]: The `==` here could be marked as a jshint error (I'm also trying to review #155), so we could use `!XKit.extensions.read_posts.undimmed_post` instead.

Comment (blackjackkent) [inline Extensions/read_posts.js]: Ah, good call, good call. :) Updated.

**Comment by bvtsang** [comment]
Looks good to me!

**Pull request merged**